packages/libs/react-ui/src/components/NavHeader/NavHeader.css.ts

```diff
@@ -130,3 +130,99 @@ export const glowClass = style([
     zIndex: 0,
   },
 ]);
+
+export const selectContainerClass = style([
+  sprinkles({
+    alignItems: 'stretch',
+    backgroundColor: {
+      lightMode: '$gray90',
+      darkMode: '$gray90',
+    },
+    borderRadius: '$sm',
+    display: 'flex',
+    flexGrow: 1,
+    lineHeight: '$lg',
+    overflow: 'hidden',
+    paddingLeft: '$4',
+    paddingRight: '$2',
+    position: 'relative',
+  }),
+  {
+    border: `1px solid ${vars.colors.$gray40}`,
+    selectors: {
+      '&:active': {
+        color: vars.colors.$gray40,
+      },
+    },
+  },
+]);
+
+export const selectContainerClassDisabled = style([
+  sprinkles({
+    backgroundColor: {
+      lightMode: '$gray20',
+    },
+    color: {
+      lightMode: '$foreground',
+    },
+  }),
+]);
+
+export const selectIconClass = style([
+  sprinkles({
+    alignItems: 'center',
+    display: 'flex',
+    marginRight: '$2',
+    color: '$gray40',
+  }),
+  {
+    selectors: {
+      '&:active': {
+        color: vars.colors.$gray40,
+      },
+    },
+  },
+]);
+
+export const selectClass = style([
+  sprinkles({
+    background: 'none',
+    border: 'none',
+    color: '$gray40',
+    flexGrow: 1,
+    outline: 'none',
+    paddingRight: '$8',
+    paddingY: '$2',
+    fontSize: '$base',
+  }),
+  {
+    backgroundColor: 'inherit',
+    appearance: 'none',
+    selectors: {
+      '&:focus-visible': {
+        borderWidth: '1px',
+        borderStyle: 'solid',
+        borderColor: vars.colors.$gray40,
+      },
+    },
+  },
+]);
+
+export const chevronIconClass = style([
+  sprinkles({
+    marginRight: '$2',
+    position: 'absolute',
+    right: '$1',
+    color: '$gray40',
+  }),
+  {
+    top: '7px',
```

Comment:
I don't think this should be absolutely. Rather I think it would be better to position the contents with flex to center. Right now I think if the height of the select changes for any reason, the alignment will be off